tests/evm_backends/pyevm_env.py

```diff
@@ -156,15 +154,15 @@ def _check_computation(self, computation):
     def get_code(self, address: str):
         return self._state.get_code(_addr(address))
 
-    def time_travel(self, num_blocks=1, time_delta: int | None = None) -> None:
+    def time_travel(self, num_blocks=1) -> None:
         """
         Move the block number forward by `num_blocks` and the timestamp forward by `time_delta`.
         """
 
         # Cast since ExecutionContextAPI does not have the properties we need to change
         context = cast(ExecutionContext, self._state.execution_context)
         context._block_number += num_blocks
-        context._timestamp += num_blocks if time_delta is None else time_delta
+        context._timestamp += num_blocks
```

Comment:
time_travel is only used in the tests to modify block number right? i think we should not touch timestamp then and also rename to something else (`fast_forward_blocks`?)

Reply:
I first did what you suggested, but some crowdfund/auction tests relied on increased timestamps. So I reverted it:
https://github.com/vyperlang/vyper/pull/3846/commits/07c0a27921bf9c3f4dc10912ba243ea3d69e21c2